### PR TITLE
pass dest project id

### PR DESCRIPTION
**Pull request by yonghaoy**
The current `IsUsingCompressedReferences`
It does not pass project id that hosted dataset. 
When the project_id is missing in a BigQuery SQL query, the bq command will use the --project_id flag specified in the command as the default project for resolving dataset and table references.
Add additional parameter to allow passing dest project. 

In our case
Error we saw in GCP console:
```
Access Denied: Table terra-vpc-sc-dev-7ee328ad:1kg_wgs_2022q1.INFORMATION_SCHEMA.COLUMNS: User does not have permission to query table terra-vpc-sc-dev-7ee328ad:1kg_wgs_2022q1.INFORMATION_SCHEMA.COLUMNS, or perhaps it does not exist.
```

`terra-vpc-sc-dev-7ee328ad:1kg_wgs_2022q1.INFORMATION_SCHEMA.COLUMNS` is wrong - `terra-vpc-sc-dev-7ee328ad` is the user workspace
It should be `fc-aou-cdr-synth-test-2.1kg_wgs_2022q1` - `fc-aou-cdr-synth-test-2` is the project that contains CDR data.

### CODE REVIEW

File: scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl [hunk 1]
```diff
@@ -78,7 +78,8 @@ workflow GvsExtractAvroFilesForHail {
 
     call Utils.IsUsingCompressedReferences {
         input:
-            project_id = project_id,
+            query_project_id = project_id,
+            dest_project_id = project_id,
```
Comment (yonghaoy) [inline scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl:82]: I did not see Hail contains dest project and I don't know the use case. So I just pass the same project

Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl:82]: Can we make one of the project_ids optional then? I dont imagine we will ever extract avro files into another project

Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl:82]: actually this is fine--let's just get it working!

File: scripts/variantstore/wdl/GvsUtils.wdl [hunk 2]
```diff
@@ -811,7 +812,7 @@ task IsUsingCompressedReferences {
       SELECT
         column_name
       FROM
-        `~{dataset_name}.INFORMATION_SCHEMA.COLUMNS`
+        `~{dest_project_id}.~{dataset_name}.INFORMATION_SCHEMA.COLUMNS`
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsUtils.wdl:815]: lovely

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/wdl/GvsUtils.wdl (hunk 2 above)
Comment (yonghaoy) [reply scripts/variantstore/wdl/GvsUtils.wdl:815]: actually I don't know wdl. Is this syntax correct?

Comment (mcovarr) [reply scripts/variantstore/wdl/GvsUtils.wdl:815]: Why is `query_project_id` being introduced here if it's never used? Possibly related, line 811 is still referencing `project_id` which no longer exists.

`womtool validate` will check the syntactical correctness of WDL for you, and is among the actions that execute automatically against `ah_var_store` PRs in this repo. I'm not sure why it didn't flag this error. 🤔

**Review by mcovarr — CHANGES_REQUESTED** [review]

File: scripts/variantstore/wdl/GvsUtils.wdl (hunk 2 above)
Comment (yonghaoy) [reply scripts/variantstore/wdl/GvsUtils.wdl:815]: Ah, good finding! I don't have full context about this, this is just a sample PR to explain my finding. 
In our scenario, user has bqJobUser role on `query_project_id` - Terra workspace project. User does not have permission on dest_project.

**Review by mcovarr — COMMENTED** [review]
I've manually kicked off our integration tests on this branch. If that passes (it should take a couple of hours), I'll give this a thumb.

**Comment by yonghaoy** [comment]
> I've manually kicked off our integration tests on this branch. If that passes (it should take a couple of hours), I'll give this a thumb.

SG! I also pushed this to Agora and will test it out using our genomic extraction workflow

**Comment by yonghaoy** [comment]
> > I've manually kicked off our integration tests on this branch. If that passes (it should take a couple of hours), I'll give this a thumb.
> 
> SG! I also pushed this to Agora and will test it out using our genomic extraction workflow

our error is resolved with this PR

**Review by mcovarr — APPROVED** [review]

**Pull request merged**